### DISCUSSION TITLE
Why are my commits not showing on my profile?

### DISCUSSION THREAD

**Discussion by josahe** [community/community#21957] [New to GitHub]
<p>I am making commits but they are not showing up on my profile. Can anyone help with why?</p>
<p>Here is my profile. It shows that that last contribution I made was on 9th December 2017:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/6331a51e7ea9841d47cfba5f98426f6ae629a689.png" data-download-href="/uploads/short-url/e9vDh8b1D327NknPdFzR6zpYsIx.png?dl=1" title="githubprofile.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25433209/181067649-b9f97b56-59f7-4469-9f95-1e1e1e1a471f.png" alt="githubprofile.PNG" data-base62-sha1="e9vDh8b1D327NknPdFzR6zpYsIx" width="637" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/6331a51e7ea9841d47cfba5f98426f6ae629a689_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">githubprofile.PNG</span><span class="informations">762×598 25.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Example of commits in past couple of days: <br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/14c746294a745e7888987a9ff0ced09497118cdc.png" data-download-href="/uploads/short-url/2XOuvKhh7l9U1JF4Mr0iX0hzzYg.png?dl=1" title="commits.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/25433209/181067650-c427fab0-e4fb-46d3-b2f3-62b1a3aa1d91.png" alt="commits.PNG" data-base62-sha1="2XOuvKhh7l9U1JF4Mr0iX0hzzYg" width="357" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/14c746294a745e7888987a9ff0ced09497118cdc_2_357x499.png, https://aws1.discourse-cdn.com/github/optimized/1X/14c746294a745e7888987a9ff0ced09497118cdc_2_535x748.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/14c746294a745e7888987a9ff0ced09497118cdc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/14c746294a745e7888987a9ff0ced09497118cdc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">commits.PNG</span><span class="informations">678×947 63.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by canuckjacq** [comment]
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git &amp; GitHub board.</p>
<p>Additionally, if you have questions that are specific to your own personal GitHub account, your best bet for an answer is to get in touch with our support team directly. You can find us <a href="https://github.com/contact" rel="nofollow noopener">here</a>, or email us at <a href="mailto:support@github.com">support@github.com</a>. </p>

**Comment by josahe** [comment]
<p>Okay. Thank you for your help.</p>

**Comment by josahe** [comment, marked as answer]
<p>I contacted support who referred me to the below page.</p>
<p><strong>Commits to a fork do not count towards your contributions.</strong></p>
<p><a href="https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/#commit-was-made-in-a-fork" rel="nofollow noopener">https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/#commit-was-made-in-a-fork</a></p>

**Comment by nadiajoyce** [comment]
<p>Hello <a class="mention" href="/josahe">@josahe</a>! I wanted to also mention here that one of our Support people actually wrote an article on this very thing. There’s lots more info and good tips there, as well:</p>
<p><a href="https://github.community/t/getting-all-your-commits-in-your-contributions-graph/10186">https://github.community/t5/Support-Series/Getting-all-your-commits-in-your-contributions-graph/ba-p/19/jump-to/first-unread-message</a></p>
<p>Thanks for being part of the GitHub Community Forum!</p>
